### DISCUSSION TITLE
Customize date and time format in status bar

### DISCUSSION THREAD

**Discussion by davidystephenson** [regolith-linux/regolith-desktop#993] [Q&A] [answered]
In the latest version of Regolith, how do I customize the date and time format shown in the status bar? I can only find documentation for older versions.

**Comment by w18g** [comment, marked as answer]
For version 3.1 (X11),  the older documentation is still valid - using the i3xrocks.date.format2 Xresource key.

The script to display the time/date is in /usr/share/i3xrocks/scripts/time

**Reply by davidystephenson** [reply]
Thank you, I was able follow the instructions here: https://regolith-linux.github.io/regolith-site-r131/docs/howto/time-format/
